client/client.go

```diff
@@ -554,6 +557,34 @@ func (c *Client) do(req *http.Request, skipLoggingPayload bool) (*container.Cont
 			log.Printf("[DEBUG] HTTP response unique string %s %s %s", req.Method, req.URL.String(), bodyStr)
 		}
 
+		// Handle session timeout for login based requests
```

Comment:
Is this block required here?

Reply:
Do you mean the comment or the whole block of code? I am not sure. 😄

Reply:
I thought handling both might be beneficial. However, let me know if it's not necessary, I can change it.